.buildkite/scripts/health-report-tests/tests/slow-start.yaml

```
@@ -8,7 +8,8 @@ config:
     pipeline.workers: 1
     pipeline.batch.size: 1
 conditions:
-  - full_start_required: false
+  full_start_required: false
+  wait_time_in_seconds: 5
```

No need to wait for slow start scenario.
```suggestion
  wait_time_in_seconds: 0
```